x/supplier/types/message_create_claim.go

```diff
@@ -47,27 +47,27 @@ func (msg *MsgCreateClaim) GetSignBytes() []byte {
 
 func (msg *MsgCreateClaim) ValidateBasic() error {
 	// Validate the supplier address
-	_, err := sdk.AccAddressFromBech32(msg.SupplierAddress)
+	_, err := sdk.AccAddressFromBech32(msg.GetSupplierAddress())
 	if err != nil {
-		return sdkerrors.Wrapf(ErrSupplierInvalidAddress, "invalid supplierAddress address (%s)", err)
+		return sdkerrors.Wrapf(ErrSupplierInvalidAddress, "%s", msg.GetSupplierAddress())
 	}
 
 	// Validate the session header
 	sessionHeader := msg.SessionHeader
 	if sessionHeader.SessionStartBlockHeight < 1 {
```

Comment:
First session ever has a `SessionStartBlockHeight` of `0`, see https://github.com/pokt-network/poktroll/blob/main/x/session/keeper/session_hydrator_test.go#L71